testsuite/shared/src/main/java/org/wildfly/test/stabilitylevel/StabilityServerSetupSnapshotRestoreTasks.java

```diff
@@ -42,6 +41,7 @@
  * the {@code teardown()} method
  */
 public abstract class StabilityServerSetupSnapshotRestoreTasks implements ServerSetupTask {
+    private static final String SERVER_ENVIRONMENT ="server-environment";
```

Comment:
Format - missing space.

Reply:
I don't have a laptop with me to fix that so I won't be able to do that on Tuesday. 🙂